fedbiomed/node/secagg/_secagg_setups.py

```diff
@@ -19,11 +19,12 @@
 from fedbiomed.common.mpc_controller import MPCController
 from fedbiomed.common.secagg import DHKey, DHKeyAgreement
 from fedbiomed.common.synchro import EventWaitExchange
+from fedbiomed.common.utils import get_default_biprime
 
 from fedbiomed.transport.controller import GrpcController
 
 from fedbiomed.node.environ import environ
-from fedbiomed.node.secagg_manager import SKManager, BPrimeManager, DHManager, SecaggManager
+from fedbiomed.node.secagg_manager import SKManager, DHManager, SecaggManager
```

Comment:
in this module also we need to update type of `experiment_id` from `Union[str, None]` to `str`